src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+public class DukeException extends Exception {
+
+  public DukeException(String message) {
```

Comment:
According to the **stupid** [Coding standard](https://se-education.org/guides/conventions/java/basic.html#layout), should it be better to indent this line as 4 spaces instead of 2?